cdec/installation-manager-core/src/test/java/com/codenvy/im/managers/TestConfigManager.java

```diff
@@ -156,7 +160,8 @@ public void testLoadDefaultCdecConfigLoadError() throws Exception {
         Path properties = Paths.get("target/test.properties");
         FileUtils.write(properties.toFile(), "a=1\n" +
                                              "b=2\n");
-        doReturn(properties).when(transport).download(endsWith("codenvy-multi-server-properties/3.1.0"), any(Path.class));
+        doReturn(properties).when(transport).download(endsWith("codenvy-multi-server-properties/3.1.0"), any(Path.class),
+                                                      MediaType.APPLICATION_OCTET_STREAM);
```

Comment:
Leads to _Mockito_ error `Unfinished stubbing detected`.
Fixup: `eq(MediaType.APPLICATION_OCTET_STREAM)`